addons/captives/functions/fnc_canBlindfoldCaptive.sqf

```diff
@@ -0,0 +1,28 @@
+#include "script_component.hpp"
+/*
+ * Author: mrschick
+ * Tests if caller has a Contact DLC blindfold in their inventory to put on the captive
+ *
+ * Arguments:
+ * 0: caller (player) <OBJECT>
+ * 1: target <OBJECT>
+ *
+ * Return Value:
+ * The return value <BOOL>
+ *
+ * Example:
+ * [player, bob] call ACE_captives_fnc_canBlindfoldCaptive
+ *
+ * Public: No
+ */
+
+params ["_unit", "_target"];
+//Alive, handcuffed, not being escorted, and caller has a Contact DLC blindfold in their inventory
+
+(_target getVariable [QGVAR(isHandcuffed), false]) &&
+{isNull (attachedTo _target)} &&
+{alive _target} &&
+{(vehicle _unit) == _unit} &&
+{(vehicle _target) == _target} &&
```

Comment:
```suggestion
{isNull objectParent_unit} &&
{isNull objectParent _target} &&
```

Reply:
I took the Captive functions "canEscortCaptive" and "doEscortCaptive" as a template for the 2 respective blindfold functions, maybe some of these changes should also be applied to the originals?

Reply:
For some reason implementing this change causes the whole interaction to not work, it gets shown regardless of the `canBlindfoldCaptive` return value (maybe it's not called) and does not run `doBlindfoldCaptive` when interacted on.